### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by isimluk**
Addressing container build failures related to dockerhub rate limiting

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
```
Comment (redhatrises) [inline Dockerfile:1]: Makes sense. To me, @shawndwells should this be a rhel8 ubi image or do we need to run specifically on 7?

Comment (isimluk) [reply Dockerfile:1]: Agreed, though strictly speaking unrelated to this change.

Comment (redhatrises) [reply Dockerfile:1]: true

**Comment by redhatrises** [comment]
Ack for this change

File: Dockerfile (hunk 1 above)
Comment (shawndwells) [reply Dockerfile:1]: Doesn't matter to me

**Review by shawndwells — APPROVED** [review]

**Pull request merged**